### PR TITLE
FM-91: Add cost centers for the facility formatter

### PR DESCRIPTION
**Pull request by leeallen337**
## Why?

Need to see the cost centers linked to a facility

https://ndustrialio.atlassian.net/browse/FM-91

## What changed?
- `Facilities#getAllByOrganizationId` to accept parameters to include cost centers information
- `Facilities#get` to include cost centers information for that specific facility

### CODE REVIEW

File: src/facilities/index.js [hunk 1]
```diff
@@ -267,6 +267,12 @@ class Facilities {
       );
     }
 
+    if (!isPlainObject(options)) {
```
Comment (zachary-kuhn) [inline src/facilities/index.js]: What if passed undefined? It's usually OK to not pass an options object to a call for most libraries.

File: src/utils/facilities/formatFacilityOptionsToServer.spec.js [hunk 2]
```diff
@@ -29,6 +31,32 @@ describe('utils/facilities/formatFacilityOptionsToServer', function() {
     expect(options).to.deep.equal(expectedOptions);
   });
 
+  it('provides an object that includes the `include_cost_centers` that the API service can understand', function() {
```
Comment (zachary-kuhn) [inline src/utils/facilities/formatFacilityOptionsToServer.spec.js]: Maybe missing a word? This didn't read quite fluidly to me.

Comment (leeallen337) [reply src/utils/facilities/formatFacilityOptionsToServer.spec.js]: 👍 

Added a word that hopefully helps

File: src/facilities/index.js (hunk 1 above)
Comment (greg5green) [reply src/facilities/index.js]: I agree with this comment.

Comment (zachary-kuhn) [reply src/facilities/index.js]: Default parameter is good! Needs less error checking that way.

**Comment by leeallen337** [comment]
@zachary-kuhn & @greg5green if you don't have anymore comments could you approve this?

**Pull request merged**